src/explore-education-statistics-admin/src/pages/release/data/components/ApiDataSetNewLocationsTable.tsx

```diff
@@ -0,0 +1,44 @@
+import { NewLocationCandidate } from '@admin/pages/release/data/utils/getApiDataSetLocationMappings';
+import locationLevelsMap from '@common/utils/locationLevelsMap';
+import React from 'react';
+import camelCase from 'lodash/camelCase';
+
+interface Props {
+  level: string;
+  locations: NewLocationCandidate[];
+}
+
+export default function ApiDataSetNewLocationsTable({
+  level,
+  locations,
+}: Props) {
+  return (
+    <table data-testid={`new-${level}`}>
+      <caption className="govuk-visually-hidden">
+        {`Table showing new locations for ${
+          locationLevelsMap[camelCase(level)]?.plural ?? level
+        }`}
+      </caption>
+      <thead>
+        <tr>
+          <th className="govuk-!-width-one-third">Current data set</th>
+          <th className="govuk-!-width-one-third">New data set</th>
+        </tr>
+      </thead>
+      <tbody>
+        {locations.map(({ candidate }) => {
+          return (
+            <tr key={`candidate-${candidate.code}`}>
+              <td>Not applicable to current data set</td>
+              <td>
+                {candidate.label}
+                <br />
+                {candidate.code}
```

Comment:
Note that there may be several codes to choose from depending on the level of the Location (e.g. `code` and `oldCode` for Local Authorities, `ukprn` for Providers etc, so there will possibly be login already in the front end code to find the appropriate codes to show based on the Location level.

Reply:
I can't find anywhere in the frontend where we currently handle this. Do you have a list of what it should be for each level?